src/features/account/components/security-reports/SecurityReportsTabPanel.tsx

```diff
@@ -0,0 +1,8 @@
+import { SimpleGrid } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+
+export const SecurityReportsTabPanel: FunctionComponent = () => {
+  return (
+    <SimpleGrid spacing="4" data-testid="security-reports-tab"></SimpleGrid>
```

Comment:
```suggestion
    <SimpleGrid spacing="4" data-testid="security-reports-tab" />
```

Reply:
it will have more components in the up coming story, for demo purpose , we leave it empty